app/views/errors/_missing_partial.html.erb

```diff
@@ -1 +1 @@
-Partial for <%= locale %> was not found, please review your application's customization and add <%= "_filename_#{locale}.html.erb" %> to the appropriate views directory.
+Partial for <%= locale %> was not found, please review your application's customization and add <%= "#{@missing_partial}.html.erb" %>.
```

Comment:
This had me confused for a little bit. It's missing the leading underscore `<%= "_#{@missing_partial}.html.erb" %>`